requirements/base.in

```diff
@@ -2,11 +2,11 @@
 
 Django>=1.11
 djangorestframework>=3.2.0,<3.10
-djangorestframework-jwt>=1.7.2,<2.0.0
+drf-jwt<1.15.0
```

Comment:
Please move the version constraint to `constraints.txt` with an explanatory comment, and leave just `drf-jwt` here.

Reply:
1.15.0 causes django22 test failures. will try to fix them tomorrow.